src/sorcha/modules/PPCalculateApparentMagnitudeInFilter.py

```diff
@@ -46,27 +53,31 @@ def PPCalculateApparentMagnitudeInFilter(
 
     H_col = "H_filter"
 
-    # first, get H, r, delta and alpha as ndarrays
-    # r, delta and alpha are converted to au from kilometres
-    r = padain["AstRange(km)"].values / 1.495978707e8
+    # first, get H, rho, delta and alpha as ndarrays
+    # delta, rho and alpha are converted to au from kilometres
+    delta = (padain["AstRange(km)"].values * u.km).to(u.au).value
 
     try:
-        delta = padain["Ast-Sun(km)"] / 1.495978707e8
+        rho = (padain["Ast-Sun(km)"].values * u.km).to(u.au).value
     except KeyError:
-        delta = (
-            np.sqrt(
-                padain["Ast-Sun(J2000x)(km)"].values ** 2
-                + padain["Ast-Sun(J2000y)(km)"].values ** 2
-                + padain["Ast-Sun(J2000z)(km)"].values ** 2
+        rho = (
+            (
+                np.sqrt(
+                    padain["Ast-Sun(J2000x)(km)"].values ** 2
+                    + padain["Ast-Sun(J2000y)(km)"].values ** 2
+                    + padain["Ast-Sun(J2000z)(km)"].values ** 2
+                )
+                * u.km
             )
-            / 1.495978707e8
+            .to(u.au)
+            .value
         )
 
     alpha = padain["Sun-Ast-Obs(deg)"].values
     H = padain[H_col].values
 
     # calculating light curve offset
-    if lightcurve_choice and LC_METHODS.get(lightcurve_choice, False):
```

Comment:
@astronomerritt was removing this part of the conditional an intentional change? I had just recently added it, but I'm curious if I missed an edge case when checking the code.

Reply:
I don't think it was intentional. I missed the lightcurve_choice was taken out. Steph was moving stuff around with light curves that I told them to put back and let Pedro update. So I think it happened there.

Reply:
Oh goodness. I think I may have messed up slightly when merging your changes into my branch. My apologies, and thank you for catching it! Please do feel free to change it back, or I can do it tomorrow.

Reply:
@drewoldag can you add it back in?

Reply:
Yes, of course. Not a problem at all :) When there are lots of moving pieces, it can be hard to keep everything organized.